add-ons/openwhisk/openwhisk.addon

```diff
@@ -0,0 +1,27 @@
+###################################################################################################
+# Name: openwhisk 
+# Description: Installs OpenWhisk FaaS 
+# Required-Vars: OPENSHIFT_USERNAME
+# OpenShift-Version: >=3.9.0       
+# Var-Defaults: OPENSHIFT_USERNAME=developer,CONSOLE_URL=https://#{ip}:8443,PROJECT_NAME=openwhisk
+# Url: Template from https://git.io/openwhisk-template 
+###################################################################################################
+
+# login
+oc login -u admin -p admin 
```

Comment:
@VeerMuchandi if you don't put login info then by default all the `oc` command in addon will run as `system:admin` and that will do needful I think.